extension/goplugin/schemas.go

```diff
@@ -683,6 +683,20 @@ func (schema *Schema) RegisterType(typeValue interface{}) {
 	schema.env.RegisterType(schema.raw.ID, typeValue)
 }
 
+// Children returns list of schemas that extend schema with given id
+func (schema *Schema) Children() []goext.ISchema {
```

Comment:
confusing naming, since parent-child relationship in gohan is already reserved for special foreign key case. Inheritance (using extend) should follow different naming.